fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -648,40 +694,179 @@ public void testCreateVersionOfBinaryWithBody() throws Exception {
         }
     }
 
-    @Ignore("Disable until binary version creation from body implemented")
     @Test
     public void testCreateVersionOfBinaryWithDatetimeAndBody() throws Exception {
         createVersionedBinary(id);
 
         final String mementoUri = createMemento(subjectUri, MEMENTO_DATETIME,
-                OCTET_STREAM_TYPE, BINARY_UPDATED);
+                "text/plain", BINARY_UPDATED);
         assertMementoUri(mementoUri, subjectUri);
 
         // Verify that the memento has the updated binary
         try (final CloseableHttpResponse response = execute(new HttpGet(mementoUri))) {
             assertMementoDatetimeHeaderMatches(response, MEMENTO_DATETIME);
 
+            // Content-type is not retained for a binary memento created without description
+            assertEquals(OCTET_STREAM_TYPE, response.getFirstHeader(CONTENT_TYPE).getValue());
+
             assertEquals("Binary content of memento must match updated content",
                     BINARY_UPDATED, EntityUtils.toString(response.getEntity()));
         }
     }
 
     @Test
-    public void testGetUnversionedObjectVersionProfile() {
-        final String pid = getRandomUniqueId();
+    public void testCreateVersionOfBinaryDescription() throws Exception {
+        createVersionedBinary(id);
+
+        final String descriptionUri = subjectUri + "/fcr:metadata";
+
+        final String mementoUri = createContainerMementoWithBody(descriptionUri, null);
+        assertMementoUri(mementoUri, descriptionUri);
+
+        setDescriptionProperty(id, null, DC.title.getURI(), "Updated");
+
+        try (final CloseableDataset dataset = getDataset(new HttpGet(mementoUri))) {
+            final DatasetGraph results = dataset.asDatasetGraph();
+
+            final Node mementoSubject = createURI(subjectUri);
+
+            assertFalse("Property added to original must not appear in memento",
```

Comment:
It would be helpful to have at least one `assertTrue` test to ensure that some expected triples are present.

Reply:
Ah I think i had that in one of my three branches... will add